linera-core/src/unit_tests/client_tests.rs

```diff
@@ -308,7 +308,7 @@ where
     assert!(sender.key_pair().await.is_ok());
     assert_eq!(
         builder
-            .check_that_validators_have_certificate(sender.chain_id, BlockHeight::from(0), 4)
+            .check_that_validators_have_certificate(sender.chain_id, BlockHeight::from(0), 3)
```

Comment:
Why this change

Reply:
It's a quick CI fix [fix flaky assert in test_rocksdb_share_ownership](https://github.com/linera-io/linera-protocol/pull/703/commits/4d2460c4f7e3ff498796dbe31bff4d32534efdf4)